### DISCUSSION TITLE
Site not refreshing

### DISCUSSION THREAD

**Discussion by ZahraOsman** [CollectionBuilder/collectionbuilder.github.io#67] [Q&A]
Hi! I've followed the tutorials and I'm at the end of the "configure your site" section. I just finished uploading my metadata file, objects, and updated my site settings to point it to the metadata file. However, my site link isn't updating from the default information. I did my best to double check my metadata file, config yml, and that my page settings are correct. 

https://zahraosman.github.io/blossom/

Any help is appreciated!

**Comment by evanwill** [comment]
@ZahraOsman your metadata CSV files "_data/blossom.csv" and "_data/plumblossom.csv" have corrupted non-UTF-8 characters in them that are causing Ruby to break, so Jekyll can not build the site. This is causing the build of your site to fail. You can tell by looking at your [Commit history](https://github.com/ZahraOsman/blossom/commits/main) you will see a red X when it fails, and a green check when it is successful. 
When the site build fails, GitHub Pages leaves up the last successful build--which is why yours isn't changing right now!

Corrupted character issues are hard to figure out... it looks like the corruption is in your "location" column. The most common way this happens is by opening or creating your CSV using Excel. Excel won't work! 
We generally suggest using Google Sheets, downloading as CSV, and adding to your repository *without* opening the file on your computer. Here is the docs describing the process: https://collectionbuilder.github.io/cb-docs/docs/metadata/uploading/

**Reply by evanwill** [reply]
p.s. @ZahraOsman everything else in your collection looks correct--once you fix the encoding issue in the "location" column, it **will** work!

**Reply by evanwill** [reply]
one more note--you will have to fix both CSVs "_data/blossom.csv" and "_data/plumblossom.csv" or delete one of them--if *any* corrupted metadata is in your "_data" folder it will break Jekyll, even if you aren't using that metadata.

**Reply by ZahraOsman** [reply]
YOU ARE THE BEST THANK YOU. 

Follow up questions: 1) should I just create a new metadata file in googlesheets and 2) once done, do I delete the other two metadata files that are corrupted? 

I just want to make sure I get the sequence right.

**Reply by evanwill** [reply]
@ZahraOsman is your current metadata in Sheets or just on your local computer? 

If it is on Sheets, you just need to [download a fresh CSV and upload](https://collectionbuilder.github.io/cb-docs/docs/metadata/uploading/). 

If it is only on your local computer, in Sheets, with a new sheet open, click File > Import to add the file from your computer to Sheets (basically follow the docs for [updating metadata](https://collectionbuilder.github.io/cb-docs/docs/metadata/uploading/#update-metadata). 

And yes, delete the two corrupted CSV files!

**Reply by ZahraOsman** [reply]
Boom. We're in there like swimwear. I appreciate all the help!
